tests/common/devices/multi_asic.py

```diff
@@ -57,8 +57,16 @@ def critical_services_tracking_list(self):
             [list]: list of the services running the device
         """
         service_list = []
-        service_list+= self._DEFAULT_SERVICES
-        for asic in self.asics:
+        active_asics = self.asics
+        if self.sonichost.is_supervisor_node() and self.get_facts()['asic_type'] != 'vs':
+            active_asics = []
+            redis_out = self.command("redis-cli -h 10.0.5.16 -p 6380 -n 13  keys \"CHASSIS_ASIC_TABLE|asic*\"")
```

Comment:
Good to remove the hardcoding for IP address, port etc in redis-cli command.  I have seen similar usage here "tests/common/helpers/voq_lag.py" as well.
 
Also would prefer to use "sonic-db-cli" instead of redis-cli.

Reply:
I have pushed a commit to remove the hard-coded 10.0.5.16 address, and get it from /etc/sonic/chassisdb.conf file.

@judyjoseph  - With regards to using sonic-db-cli, I don't see an option in sonic-db-cli for host and port. Would you know the equivalent sonic-db-cli command?

Looked at the example pointed in tests/common/helpers/voq_lag.py, but that uses VoqDbCli which is using redis-cli underneath. I could change to use the VoqDbCli class instead if you want.